src/main/scala/com/suprnation/actor/debug/TrackingActor.scala

```diff
@@ -146,8 +148,11 @@ final case class TrackingActor[F[+_]: Parallel: Concurrent: Temporal, Request, R
 ) extends ReplyingActor[F, Request, Response]
     with ActorConfig {
 
-  override val receive: ReplyingReceive[F, Request, Response] = { case m =>
-    messageBufferRef.update(_ :+ m) >> proxy.receive(m)
+  override val receive: ReplyingReceive[F, Request, Response] = {
+    case _ @ Timer(_, m, _, _, _) =>
+      messageBufferRef.update(_ :+ m) >> proxy.receive(m.asInstanceOf[Request])
```

Comment:
This works, but I think we can have a more robust solution by having `TrackingActor` extend `Timers`. What do you think?